tooling/minimize/src/enums.rs

```diff
@@ -177,6 +177,10 @@ impl<'tcx> Ctxt<'tcx> {
             .collect()
     }
 
+    pub fn discriminant_for_variant_stable(&self, ty: smir::Ty, variant_idx: smir::VariantIdx) -> Int {
+        self.discriminant_for_variant(smir::internal(self.tcx, ty), smir::internal(self.tcx, variant_idx))
```

Comment:
Ah, there's a way to go from an smir type back to its nternal type? That's good to know.

Reply:
Yes, many constructs in SMIR can be translated back. The main exception today is MIR body components, like statements, operand, rvalue.